### DISCUSSION TITLE
如果在K8s部署多个KyuubiServer，如何进行会话保持？

### DISCUSSION THREAD

**Discussion by shenbinglife** [apache/kyuubi#4421] [Q&A]
假如启动一个3副本的KyuubiServer Deployment，客户端怎么保证连接到同一个Kyuubi Server实例 ？

**Comment by pan3793** [comment]
This is no issue for `THRIFT_BINARY` since it uses a long-run TCP socket, for short connections like HTTP, you may need "session sticky" feature. Ref: https://docs.ovh.com/asia/en/kubernetes/sticky-session-nginx-ingress/
